fe/fe-core/src/main/java/org/apache/doris/nereids/cost/CostModelV1.java

```diff
@@ -262,6 +265,20 @@ public Cost visitPhysicalHashJoin(
 
         double leftRowCount = probeStats.getRowCount();
         double rightRowCount = buildStats.getRowCount();
+        // if the diff of the left side and right side row count is
+        // less than this threshold, we will consider their connectivity
+        double ignorableDiffBetweenLeftAndRightThreshold = 10.0;
+        if (Math.abs(leftRowCount - rightRowCount) < ignorableDiffBetweenLeftAndRightThreshold
```

Comment:
better to restrict this corn case to left row count equals to right count

Reply:
done